command/secrets_tune_test.go

```diff
@@ -166,6 +166,9 @@ func TestSecretsTuneCommand_Run(t *testing.T) {
 				"-max-lease-ttl", "1h",
 				"-audit-non-hmac-request-keys", "foo,bar",
 				"-audit-non-hmac-response-keys", "foo,bar",
+				"-passthrough-request-headers", "authorization",
```

Comment:
Are you planning to add checks below to verify that these arguments had an effect?

Reply:
Yes, Done